### ISSUE TITLE
Add language declaration to HTML header

### ISSUE THREAD

**Issue by brainwane** [solutionguidance/psm#517]
`pa11y` reminds us that, per `WCAG2AA.Principle3.Guideline3_1.3_1_1.H57.2`, The `html` element should have a `lang` or `xml:lang` attribute which describes the language of the document.

Example of current code:
~~~
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"><head>
  <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
~~~

Success Criterion:
[3.1.1: Language of Page](http://www.w3.org/TR/WCAG20/#meaning-doc-lang-id)
Suggested Techniques:
[H57](http://www.w3.org/TR/WCAG20-TECHS/H57)
